ckanext/switzerland/plugins.py

```diff
@@ -136,6 +136,8 @@ def get_helpers(self):
             'get_group_count': ogdch_frontend_helpers.get_group_count,
             'get_localized_org': ogdch_frontend_helpers.get_localized_org,
             'localize_json_facet_title': ogdch_frontend_helpers.localize_json_facet_title, # noqa
+            'localize_harvester_facet_title': ogdch_backend_helpers.localize_harvester_facet_title, # noqa
+            'localize_showcase_facet_title': ogdch_backend_helpers.localize_showcase_facet_title, # noqa
```

Comment:
I couldn't put these in `frontend_helpers` like the `localize_json_facet_title` function, because it led to a circular dependency. Now that we no longer have frontend tempates in this ckanext, we should probably combine/refactor those helper files, but that's outside the scope of this issue.